stack.yaml

```diff
@@ -4,8 +4,13 @@ extra-deps:
   - haskell-src-exts-1.21.0
   - haskell-src-exts-util-0.2.5
   - archive: https://digitalassetsdk.bintray.com/ghc-lib/ghc-lib-parser-8.8.1.20191204.tar.gz
+  - github: shayne-fletcher/ghc-lib-parser-ex
```

Comment:
Why do we need to get this from your git repo given that you uploaded this to hackage?

Reply:
Actually, you have a point, it's shouldn't be necessary to go to git. Let me test and assuming it works out, I'll fix.

Reply:
Yep, fixed.